### DISCUSSION TITLE
reverse selecting

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [saulpw/visidata#1393] [Q&A] [answered]
just found out I missed toggle.
sorry for spam.
You can delete this thread.
cleared

**Comment by anjakefala** [comment, marked as answer]
@Shahin-rmz, this is not spam!! It is useful to leave up Q + A for future folks, and for us to learn about non-intuitive wording.

The original question was: 

Hello,
I was wondering, there is any possibilities to select the unselected rows, and unselect the selected rows?
Thanks

And the answer is `toggle-row`, by default bound to `t`. `t` will switch the selection of the current row. `gt` will switch the selection of all rows.
